### ISSUE TITLE
evaluate_model.py: ValueError when number of samples > 10000

### ISSUE THREAD

**Issue by folivetti** [EpistasisLab/regression-benchmark#17]
In this line of code I'm getting the error ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all():

https://github.com/EpistasisLab/regression-benchmark/blob/967c66beec7e79ef53d6a3bb133190887224cfc1/experiment/evaluate_model.py#L36

I suggest changing it to:

```python
sample_idx = np.random.choice(np.arange(len(labels)), size=n_samples)
```

**Comment by lacava** [comment]
should be fixed in 884bb83

**Comment by lacava** [comment]
or rather, fixed in deb3cce28fe879fdd8adfeeef58f237ecfa4b671.
